style/input.less

```diff
@@ -0,0 +1,6 @@
+*[ dir='rtl' ] {
+	input[ nav-selected='true' ] {
```

Comment:
I'm confused by the `nav-selected` here... The position of the icon changes when it becomes selected?

Reply:
Yes, the icon shown only when the input is selected.